fix(cdp): update batch export instructions (#10208)
PostHog/posthog.com: contents/docs/cdp/batch-exports/snowflake.md

## Creating the batch export

1. Subscribe to data pipelines add-on in [your billing settings](https://us.posthog.com/organization/billing) if you haven't already.

> 2. Click [Data pipelines](https://app.posthog.com/pipeline) in the navigation and go to the destinations tab in your PostHog instance.
> 3. Click "New destination".
> 4. Select **Snowflake** as the batch export destination.

Reviewer: ```suggestion
2. Click [Data pipelines](https://app.posthog.com/pipeline) in the navigation and go to the **Destinations** tab in your PostHog instance.
3. Search for **Snowflake**.
4. Click the **+ Create** button. 
```
